src/components/ui/custom/navigationButton.tsx

```
@@ -20,11 +30,11 @@ export default function NavigationButtons({
 
   return (
     <div className="fixed right-0 bottom-0 left-0 flex justify-center bg-background/80 p-4 backdrop-blur-sm">
-      <div className="mx-auto flex w-full max-w-md justify-between px-4 py-4">
-        <Button type="button" onClick={prev}>
+      <div className="mx-auto flex w-full max-w-md justify-between  py-4">
```

Biome (1.9.4)

[error] These CSS classes should be sorted.

Safe fix: Sort the classes.


(lint/nursery/useSortedClasses)